[2024-11-12T01:32:08.893+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-12T01:32:09.103+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_To_Data_Warehouse.clear_data_to_stgNCC manual__2024-11-12T01:27:51.231896+00:00 [queued]>
[2024-11-12T01:32:09.112+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_To_Data_Warehouse.clear_data_to_stgNCC manual__2024-11-12T01:27:51.231896+00:00 [queued]>
[2024-11-12T01:32:09.112+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-12T01:32:09.127+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): clear_data_to_stgNCC> on 2024-11-12 01:27:51.231896+00:00
[2024-11-12T01:32:09.134+0000] {standard_task_runner.py:63} INFO - Started process 4769 to run task
[2024-11-12T01:32:09.137+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_To_Data_Warehouse', 'clear_data_to_stgNCC', 'manual__2024-11-12T01:27:51.231896+00:00', '--job-id', '648', '--raw', '--subdir', 'DAGS_FOLDER/ETL_LDTBXH.dag.py', '--cfg-path', '/tmp/tmpwxyadoh0']
[2024-11-12T01:32:09.138+0000] {standard_task_runner.py:91} INFO - Job 648: Subtask clear_data_to_stgNCC
[2024-11-12T01:32:09.187+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_To_Data_Warehouse.clear_data_to_stgNCC manual__2024-11-12T01:27:51.231896+00:00 [running]> on host 3d0ab7361361
[2024-11-12T01:32:09.286+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_To_Data_Warehouse' AIRFLOW_CTX_TASK_ID='clear_data_to_stgNCC' AIRFLOW_CTX_EXECUTION_DATE='2024-11-12T01:27:51.231896+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-12T01:27:51.231896+00:00'
[2024-11-12T01:32:09.289+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-12T01:32:09.364+0000] {logging_mixin.py:188} WARNING - /opt/***/plugins/NCC/clear_data_stgNCC.py:16 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2024-11-12T01:32:24.462+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-12T01:32:24.463+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-12T01:32:24.471+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_To_Data_Warehouse, task_id=clear_data_to_stgNCC, run_id=manual__2024-11-12T01:27:51.231896+00:00, execution_date=20241112T012751, start_date=20241112T013209, end_date=20241112T013224
[2024-11-12T01:32:24.500+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-12T01:32:24.524+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-12T01:32:24.528+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
